### ISSUE TITLE
«huao»

### ISSUE THREAD

**Issue by ciuak** [toaq/roots#19]
"huao": {
      "official": {
        "english": "▯ is loose.",
        "gloss": "loose"
      },
      "frame": "c"
    }

Definition unclear. The word hasn't been used even once. Wiktionary gives at least 7 relevant definitions:

> 1. Not fixed in place tightly or firmly.
>   > This wheelbarrow has a loose wheel.
>    
> 2. Not held or packaged together.
>   > You can buy apples in a pack, but they are cheaper loose.
>   
> 3. Not under control. 
>   > The dog is loose again.
>   
> 4. Not fitting closely
>   > I wear loose clothes when it is hot.
>   
> 5. Not compact. 
>   > It is difficult walking on loose gravel.
>   > a cloth of loose texture
>   
> 6. Relaxed.
>   > She danced with a loose flowing movement.
>   
> 7. Not precise or exact; vague; indeterminate. 
>   > a loose way of reasoning

**Comment by lynn** [comment]
I suggest 1 and 4 (and I guess 5?) as literal meanings, and the others as metaphorical meanings (I defined **huaomōe** once for feeling relaxed / stress-free.)

I've always assumed this is Lojban's `kluza`. A **húao** is something that can easily (be) move(d) around a little in its otherwise relatively fixed position. A **rúeq** (Lojban `tagji`) is something that has no such wiggle room in its fixed position.

I quite like the x2 of these Lojban words. You can think of these as position words like **buı** or **neo**. A shirt can be loose _on_ my torso while I'm wearing it, but tight _on_ someone else's body while they are wearing it. A lid is tight _on_ a bottle. So, maybe these two should have a second place for the thing they're tight on, or loose/bloused on.

Maybe then a rigid thing is then **taq rueq** and an flexible thing is **taq huao** 😮 .

**Comment by xorxes** [comment]
I don't think you need an "on" slot because you can have **rueq neo**, **rueq nie**, **rueq rei**, **rueq heq**, **rueq coe**, **rueq tie**, etc,

**Comment by lynn** [comment]
That's a good point! I might even prefer it that way.
